Documentation/gettingstarted/istio.rst

```diff
@@ -22,6 +22,15 @@ environment running on your machine.
    instructions provided here for the other GSGs. 5 GB and 4 CPUs
    should be enough for this GSG (``--memory=5120 --cpus=4``).
 
+.. note::
+
+   If Cilium is deployed in kube-proxy-free mode, you need to set
+   ``--bpf-socket-lb-hostns-only: true`` in the deployment yaml
+   directly or via ``bpf.socketHostnsOnly`` option with Helm.
+   Without this option, when Cilium does service resolution via
+   socket load balancing, Istio sidecar will be bypassed, resulting
+   in loss of Istio features including encryption and telemetry.
+
```

Comment:
This agent knob definitely also needs documentation in Documentation/gettingstarted/kubeproxy-free.rst under advanced configuration, perhaps a small section on its own there. What would also need to be documented are a clear documentation on the limitations of this approach. Which services are affected in what way if we only allow the E-W LB in hostns, see also one of my earlier comments.

Reply:
Added a section there with some details, we can iterate from there I guess :)